plasmapy/formulary/tests/test_ionization.py

```diff
@@ -3,17 +3,37 @@
 import pytest
 from astropy.constants import c
 
-from plasmapy.formulary.ionization import Z_bal
+from plasmapy.formulary.ionization import Saha, ionization_balance
 
 
-def test_Z_bal():
+def test_ionization_balance():
     n = 1e19 * u.m ** -3
     T_e = 5000 * u.K
 
-    assert (Z_bal(n, T_e) * u.dimensionless_unscaled).unit == u.dimensionless_unscaled
+    assert (
+        ionization_balance(n, T_e) * u.dimensionless_unscaled
+    ).unit == u.dimensionless_unscaled
```

Comment:
```suggestion
    assert ionization_balance(n, T_e).unit == u.dimensionless_unscaled
```

Reply:
what's the significance of the specific number in line 15?

Reply:
nevermind, i see that it's just a value test